react/components/Image.js

```diff
@@ -1,21 +1,49 @@
 import React from 'react'
 import PropTypes from 'prop-types'
+import {path} from 'ramda'
 
 import productSummary from '../productSummary.css'
 
 /** Image component with 1:1 aspect ratio */
-export default function Image({ src, className }) {
+export default function Image({ src, className, width, height, alt }) {
+  const resizeSrc = src => {
+    //Check of src is already in a size defined by the client
+    const isWithMeasures = new RegExp(/[1-9]+-(:?([1-9]+)|(auto))/);
+    if (isWithMeasures.exec(src)) {
+      return src
+    }
+
+    const imageId = path('2', src.slice(src.indexOf("/ids/")).split("/"))
+    const withSuffix = `${imageId}-${width}-${height}`
```

Comment:
Return the query string used on the new **vtex.file-manager** too.
Example: https://github.com/vtex-apps/product-details/blob/master/react/utils/generateUrl.js#L16